### DISCUSSION TITLE
Сделать автоматический выбор текущего режима, при создании комнаты

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#18] [Ideas]
Предлогаю, в поле выбора режима, писать тот режим - который находится в сейчашней вкладке, например, нажал на вкладку тдм, решил создать сервер, и сразу выбран тдм

**Comment by kkohno** [comment]
если про создание комнаты то да, об этом думали...

**Comment by kkohno** [comment]
сделано

**Reply by KukushkaSex** [reply]
Благодарю
